wdl-lint/src/rules/malformed_lint_directive.rs

```diff
@@ -0,0 +1,119 @@
+//! A lint rule for flagging malformed lint directives.
+use wdl_ast::AstToken;
+use wdl_ast::Comment;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Document;
+use wdl_ast::Span;
+use wdl_ast::SupportedVersion;
+use wdl_ast::VisitReason;
+use wdl_ast::Visitor;
+
+use crate::Rule;
+use crate::Tag;
+use crate::TagSet;
+use crate::util::is_inline_comment;
+
+/// The identifier for the Malformed Lint Directive rule.
+const ID: &str = "MalformedLintDirective";
+/// The accepted lint directives.
+const ACCEPTED_LINT_DIRECTIVES: [&str; 1] = ["except"];
+
+/// Creates an "Inline Lint Directive" diagnostic.
+fn inline_lint_directive(span: Span) -> Diagnostic {
+    Diagnostic::warning("lint directive must be on its own line")
+        .with_rule(ID)
+        .with_label("malformed lint directive", span)
+}
+
+/// Creates an "Invalid Lint Directive" diagnostic.
+fn invalid_lint_directive(span: Span) -> Diagnostic {
+    let accepted_directives = ACCEPTED_LINT_DIRECTIVES.join(", ");
+    Diagnostic::warning("lint directive not recognized")
+        .with_rule(ID)
+        .with_label("lint directive not recognized", span)
+        .with_fix(format!(
+            "use any of the recognized lint directives: [{:#?}]",
+            accepted_directives
+        ))
+}
+
+/// Creates a "Missing Whitespace" diagnostic.
+fn missing_whitespace(span: Span) -> Diagnostic {
+    Diagnostic::warning("missing whitespace before lint directive")
+        .with_rule(ID)
+        .with_label("missing whitespace before lint directive", span)
+}
+
+/// Creates a "No Colon Detected" diagnostic.
+fn no_colon_detected(span: Span) -> Diagnostic {
+    Diagnostic::warning("no colon detected following lint directive")
+        .with_rule(ID)
+        .with_label("no colon detected following lint directive", span)
+}
+
+/// Detects a malformed lint directive.
+#[derive(Default, Debug, Clone, Copy)]
+pub struct MalformedLintDirectiveRule;
+
+impl Rule for MalformedLintDirectiveRule {
+    fn id(&self) -> &'static str {
+        ID
+    }
+
+    fn description(&self) -> &'static str {
+        "Flags malformed lint directives"
+    }
+
+    fn explanation(&self) -> &'static str {
+        "Comments which begin with `#@` must only contain valid lint directives. Lint directives \
+         must be on their own line, only preceded by whitespace."
```

Comment:
This text (that I wrote and gave you 😅 ) could use a little more detail. I'll get back to you with a rewrite.

Reply:
@a-frantz pending revision/approval, I think this is the last step that remains in this PR!

Reply:
```
        "Comments which begin with `#@` must only contain valid lint directives. Lint directives \
         must be on their own line, only preceded by whitespace. Lint directives should follow the \
         pattern `#@ <directive>: <value>` _exactly_. Currently the only accepted lint directive \
         is `except`. For example, `#@ except: MalformedLintDirective`."
```